nagios/check.py

```diff
@@ -172,6 +176,27 @@ def check(self, instance):
             raise Exception('No Nagios configuration file specified')
         for tailer in self.nagios_tails[instance_key]:
             tailer.check()
+        i_key = {"type": self.INSTANCE_TYPE, "url": instance.get("nagios_conf")}
+        self.get_topology(i_key)
+
+    def get_topology(self, instance_key):
+
+        # Get all hosts
+        self.start_snapshot(instance_key)
+        all_hosts = Model.Host.objects.all
+        for host in all_hosts:
+            if host.host_name is None:
+                continue
+            id = host.host_name
+            type = {
+                "name": "nagios-host"
+            }
+            data = {
+                "name": host.host_name.strip(),
+                "tags": "nagios-server:"+id
```

Comment:
looking at other StackPacks, do we also want to include the labels here?  I've seen a similar label like 'stackpack:nagios' in other StackPacks. Or is this done in the StackState templates?

Reply:
In case we can obtain the Nagios' hostname, that might be a very good label as well, telling the user which Nagios instance monitors this host and from where the component originates from. This might then also be a good candidate for the instance url.

Reply:
> looking at other StackPacks, do we also want to include the labels here? I've seen a similar label like 'stackpack:nagios' in other StackPacks. Or is this done in the StackState templates?

I am adding the label `nagios` in the template itself.

Reply:
> In case we can obtain the Nagios' hostname, that might be a very good label as well, telling the user which Nagios instance monitors this host and from where the component originates from. This might then also be a good candidate for the instance url.

hostname label is added as `nagios-server:<name>` in the labels section.